oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,409 @@
+=============================
+OEP-0055: Project Maintainers
+=============================
+
+.. This OEP template is based on Python's PEP standard.
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - Link to the doc in the following format::
+
+        :doc:`OEP-0055 <oep-0055-proc-Project Maintainers>`
```

Comment:
This link is broken. Also nit: According to the OEP spec, you should be putting `proc` in the document name (`oep-0055-proc-project-maintainers`, see https://github.com/openedx/open-edx-proposals/tree/master/oeps/processes - all (except oep-1, and oep-54 where I also missed the notice) and https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#oep-header-preamble)